tests/resources/Nimbus-Util.robot

```diff
@@ -63,4 +63,62 @@ Create a VSAN Cluster
     Set Environment Variable  TEST_RESOURCE  cls
     Set Environment Variable  TEST_TIMEOUT  30m
     
-    Gather Host IPs
\ No newline at end of file
+    Gather Host IPs
+
+Create a Simple VC Cluster
```

Comment:
This does not seem to be getting used right now. Is this for future cluster work?